goal_src/jak1/pc/features/autosplit-h.gc

```diff
@@ -11,7 +11,41 @@
    (num-power-cells uint32)
    (num-orbs uint32)
    (num-scout-flies uint32)
-   (padding-stats uint8 200) ;; padding for future growth
+   ;; orbs per-level
+   (training-num-orbs uint32)
+   (village1-num-orbs uint32)
+   (beach-num-orbs uint32)
+   (jungle-num-orbs uint32)
+   (misty-num-orbs uint32)
+   (firecanyon-num-orbs uint32)
+   (village2-num-orbs uint32)
+   (sunken-num-orbs uint32)
+   (swamp-num-orbs uint32)
+   (rolling-num-orbs uint32)
+   (ogre-num-orbs uint32)
+   (village3-num-orbs uint32)
+   (snow-num-orbs uint32)
+   (cave-num-orbs uint32)
+   (lavatube-num-orbs uint32)
+   (citadel-num-orbs uint32)
+   ;; scout flies per-level
+   (training-num-scout-flies uint32)
+   (village1-num-scout-flies uint32)
+   (beach-num-scout-flies uint32)
+   (jungle-num-scout-flies uint32)
+   (misty-num-scout-flies uint32)
+   (firecanyon-num-scout-flies uint32)
+   (village2-num-scout-flies uint32)
+   (sunken-num-scout-flies uint32)
+   (swamp-num-scout-flies uint32)
+   (rolling-num-scout-flies uint32)
+   (ogre-num-scout-flies uint32)
+   (village3-num-scout-flies uint32)
+   (snow-num-scout-flies uint32)
+   (cave-num-scout-flies uint32)
+   (lavatube-num-scout-flies uint32)
+   (citadel-num-scout-flies uint32)
+   (padding-stats uint8 72) ;; padding for future growth
```

Comment:
we could retain a lot of the extra space by making the orb counts uint8 -- surely 255 per level will be sufficient?

Reply:
Oh duh yeah, I think I was following the pattern from game total counts. I'll shrink these